### PR TITLE
add-dropdown-example

### PR DESCRIPTION
**Pull request by Gugo-1991**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -12,13 +14,17 @@ function App() {
   const modal = useSelector(function (state) {
     return state.modal.showmodal;
   });
+  const adduser = useSelector(function (state) {
+    return state.modal.showAddusersModal;
```
Comment (agularyan) [inline src/app.js]: please change `showAddusersModal` to `showAddUsersModal`

File: src/modal/modalContent/addCard.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  const [price, setPrice] = useState("");
+  const [img, setImg] = useState("");
+  const dispatch = useDispatch();
+
+  return (
+    <>
+      <input
+        required
+        placeholder=" type name..."
+        onChange={(e) => setNAme(e.target.value)}
```
Comment (agularyan) [inline src/modal/modalContent/addCard.js]: please change `setNAme` to `setName`

Comment (Gugo-1991) [reply src/modal/modalContent/addCard.js]: ofcourse

File: src/app.js (hunk 1 above)
Comment (Gugo-1991) [reply src/app.js]: ofcourse

**Pull request merged**